robottelo/ui/role.py

```diff
@@ -77,6 +77,30 @@ def add_permission(self, role_name, resource_type=None,
             self.configure_entity(location, FILTER['filter_loc'])
         self.click(common_locators['submit'])
 
+    def get_permissions_by_filter_resource(
```

Comment:
Since it's filters page, not roles, proper name should be smth like `filters_get_permissions_by_resource`.